tmt/steps/provision/__init__.py

```diff
@@ -625,6 +625,11 @@ def is_ready(self) -> bool:
 
         raise NotImplementedError
 
+    @property
+    def logs(self) -> list[str]:
+
+        raise NotImplementedError
```

Comment:
Seems like returning an empty list would be a nice default implementation. There are also plugins outside of tmt repository, they might get updated much later, so it would be safer to return something rather than raising an exception.